### DISCUSSION TITLE
Failure - Main Log in to GitHub Container Registry

### DISCUSSION THREAD

**Discussion by Christiantyemele** [nektos/act#2555] [General]
i am on ubuntu arm64 and i have a workflow file to package my docker image to github container registry, but i have the following error which i do not know how to go about 
![Screenshot from 2024-12-04 09-27-57](https://github.com/user-attachments/assets/222ed584-ab61-4966-a8d0-dfc7d3bfd162)

**Comment by ChristopherHX** [comment]
did you provide the `GITHUB_TOKEN` secret?
